### PR TITLE
Javascript javascript1 week4

### PR DESCRIPTION
**Pull request by oleksandra-lavryk**
Javascript javascript1 week4

### CODE REVIEW

File: javascript/javascript1/week4/main.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+  if (command.indexOf(setTimerStr) >= 0) {
+    let minutes = parseInt(
+      command.replaceAll(setTimerStr, "").replaceAll(" minutes", "")
+    );
+    function timerFinished() {
+      console.log("Timer done");
+    }
+    setTimeout(timerFinished, minutes * 60 * 1000);
+    console.log("Timer set for " + minutes + " minute(s)");
+  }
```
Comment (vulk29) [inline javascript/javascript1/week4/main.js:112]: but what if i set it in seconds? :)

File: javascript/javascript1/week4/main.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+getReply("Add singing in the shower to my todo list");
+getReply("Remove reading from my todo list");
+getReply("What is on my todo?");
+getReply("What day is it today?");
+getReply("What is 12 + 52?");
+getReply("What is 12 - 2?");
+getReply("What is 12 * 10?");
+getReply("What is 12 / 2?");
+getReply("Set a timer for 1 minutes");
+getReply("How many commands have I written?");
```
Comment (vulk29) [inline javascript/javascript1/week4/main.js:160]: i like this one! really cool! :)

File: javascript/javascript1/week4/main.js [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+  commnadCounter = commnadCounter + 1;
+
+  if (command.indexOf(checkNameStr) >= 0) {
+    if (userName.length == 0) {
+      userName = command.slice(checkNameStr.length + 1, command.length);
+      console.log("Nice to meet you, " + userName + ".");
+    } else {
+      console.log("I already know your name, " + userName + " :)");
+    }
+  }
```
Comment (vulk29) [inline javascript/javascript1/week4/main.js:44]: really smart! took me a second to understand what you did here, so remember to comment your code here and there :)

**Review by vulk29 — APPROVED** [review]

File: javascript/javascript1/week4/main.js (hunk 1 above)
Comment (oleksandra-lavryk) [reply javascript/javascript1/week4/main.js:112]: @vulk29  I need to think more time in this case

File: javascript/javascript1/week4/main.js (hunk 2 above)
Comment (oleksandra-lavryk) [reply javascript/javascript1/week4/main.js:160]: @vulk29 thank you very much 😊

**Pull request merged**